.github/pull_request_template.md

```diff
@@ -0,0 +1,9 @@
+**JIRA Ticket**
+https://perigongames.atlassian.net/browse/BFB- *[TICKET NUMBER]*
```

Comment:
can you remove the space between BFB- *[TICKET....

it's too ez to make a mistake in the url